### DISCUSSION TITLE
Connection to EMQX broke, please help!

### DISCUSSION THREAD

**Discussion by AndreasFlies** [Koenkk/zigbee2mqtt#23273] [General]
After I tried to add and remove two Sonoff motion sensors because of their strange behaviour, zigbee2mqtt cannot connect completely to EMQX any longer.

![Screenshot 2024-07-06 at 15-28-47 Home Assistant](https://github.com/Koenkk/zigbee2mqtt/assets/8461354/a6372974-8487-4231-896e-101a32e86efc)

-----------

There seem to be frequent reconnections. In EMQX, I can see the number conncetions switching between 1 and 2. Until today, there have been constantly 2 connections to ther broker (zigbee2mqtt and the mqtt integration of home assistant). zigbee2mqtt seems to use a undefined ip address:
![Screenshot 2024-07-06 at 16-37-12 EMQX – Home Assistant](https://github.com/Koenkk/zigbee2mqtt/assets/8461354/c058343e-a3f5-47bd-98a5-f3d1e01ce131)

**What can I do?**

Thank you!

**Comment by mpuff** [comment]
had the same, I was going back to mosquitto, I run it in docker, so it was very easy for me.
emqx was working with v 1.38, but with v1.39, same like you.
i have no experience with emqx, did a first try, standard setting, maybe there are recommended settings for emqx, I am also interested, I like the gui of emqx, nice view if everything is connected and if there are wrong configured devices that are trying to connect without user/password and so on.

**Comment by AndreasFlies** [comment]
Thank you! I switched to mosquitto, everything works now.

EMQX was 0.6.0 (add-on for home assistant).
